src/Nethermind/Nethermind.Core/Eip7594Constants.cs

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Core;
+
+public class Eip7594Constants
+{
+    public const int MaxBlobsPerTx = 6;
```

Comment:
A link to a spec would be great for future maintainers

Reply:
Good idea, added